packages/app/components/Settings/ProfileSettings.tsx

```diff
@@ -40,76 +39,86 @@ export default function ProfileSettings(): ReactElement {
     message.success("Your profile settings have been successfully updated");
   };
 
+  useEffect(() => {
+    setMobile(window.innerWidth < 768);
+  });
+
   return profile ? (
-    <div style={{ paddingTop: "50px" }}>
-      <Form form={form} initialValues={profile}>
-        <Form.Item
-          label="First Name"
-          name="firstName"
-          data-cy="firstNameInput"
-          rules={[
-            {
-              required: true,
-              message: "Your name can't be empty!",
-            },
-          ]}
-        >
-          <Input />
-        </Form.Item>
-        <Form.Item
-          label="Last Name"
-          name="lastName"
-          data-cy="lastNameInput"
-          rules={[
-            {
-              required: true,
-              message: "Your name can't be empty!",
-            },
-          ]}
-        >
-          <Input />
-        </Form.Item>
-        {(role === Role.TA || role === Role.PROFESSOR) && (
-          <div>
+    <div>
+      <HeaderTitle>
+        <h1>Personal Information</h1>
+      </HeaderTitle>
+      <Form wrapperCol={{ span: 18 }} form={form} initialValues={profile}>
+        {isMobile ? (
+          <>
+            <Form.Item
```

Comment:
It looks like the actual form items within both branches of the ternary are the same, right? I think it might be a bit cleaner to have some sort of helper component like `ResponsiveFormRow` that renders its `children` within a `<Row>` if it is mobile and within a Fragment if not. This way you wouldn't have to duplicate the code for the form items. You could even manage the `isMobile` state within that helper component if it's only being used here.

Reply:
Is something like this fine:
```
const ResponsiveFormRow = styled(Row)`
    flexDirection: ${isMobile ? 'column' : 'row'};
  `;
```

Reply:
if it looks fine on the frontend then I say that is fine 👍